Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -597,7 +613,7 @@ extension RouteController: Router {
             return userIsWithinRadiusOfDestination(location: location)
         }
         else {
-            let offRoute = status.routeState == .offRoute || status.routeState == .invalid
+            let offRoute = status.routeState == .offRoute
```

Comment:
`status.routeState == .invalid` doesn't mean we are offroute. It means that we are still in passive navigation. See https://github.com/mapbox/mapbox-navigation-ios/pull/3432/files#r722128724